DEPS

```diff
@@ -56,8 +54,6 @@ deps = {
     (Var("chromium_git")) + '/chromium/src.git@' + (Var("chromium_version")),
   'src/third_party/electron_node':
     (Var("electron_git")) + '/node.git@' + (Var("node_version")),
-  'src/electron/vendor/pyyaml':
```

Comment:
Please keep it as an optional dependency.
It is still used to parse [unit tests configs](https://github.com/electron/electron/blob/master/spec/configs/unittests.yml).

Reply:
Not sure how to make it optional, so i just reverted the removal.

Reply:
By "optional" I meant adding a flag `checkout_pyyaml: False`.